src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -96,7 +96,7 @@ const Small: React.FC<ChannelImageBadgeProps> = ({type}) => {
 
   if (type === 'GROUP') {
     return (
-      <View style={[styles.badgeContainer, styles.containersigned_primary]}>
+      <View style={[styles.badgeContainer, styles.containersigned_secondary]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change from `containersigned_primary` to `containersigned_secondary` alters the background color of the 'GROUP' badge. Again, ensure this color change aligns with the overall design and accessibility guidelines.

```diff
- <View style={[styles.badgeContainer, styles.containersigned_primary]}>
+ <View style={[styles.badgeContainer, styles.containersigned_secondary]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->